src/frontends/onnx/frontend/src/op/col2im.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (C) 2018-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "openvino/op/col2im.hpp"
+
+#include "core/operator_set.hpp"
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace onnx {
+namespace ai_onnx {
+namespace opset_1 {
+ov::OutputVector col2im(const ov::frontend::onnx::Node& node) {
+    const auto data = node.get_ov_inputs().at(0);
+    const auto output_shape = node.get_ov_inputs().at(1);
+    
+    std::vector<int64_t> kernel_shape;
+    if (node.has_attribute("kernel_shape")) {
+        kernel_shape = node.get_attribute_value<std::vector<int64_t>>("kernel_shape");
+    } else {
+        FRONT_END_GENERAL_CHECK(false, "Col2Im requires 'kernel_shape' attribute.");
+    }
```

Comment:
Where does this "kernel_shape" attribute come from? It's not mentioned in the ONNX Col2Im specification (https://onnx.ai/onnx/operators/onnx__Col2Im.html#col2im-18).